castle/cms/profiles/2_1_0/browserlayer.xml

```diff
@@ -1,5 +1,5 @@
 <layers>
   <layer
     name="castle.cms.pwexpiry"
-    interface="castle.cms.pwexpiry.interfaces.ICollectivePWExpiryLayer" />
+    interface="castle.cms.interfaces.passwordvalidation.ICollectivePWExpiryLayer" />
```

Comment:
Can we remove the layer support?

We should only really be using ICastleLayer

Reply:
Is there a way to un-register a layer? Is it necessary to do so if it's possible?

Reply:
I think if you do `remote="True"`, it should remove it. You might need to test.

Some of those things requires you leave in the reference to the old layer class so it can load/import the definition. I'm not exactly sure if it is required when removing though.